it seems we have gcc traoubles

```
g++: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-4.6/README.Bugs> for instructions.
make[3]: *** [rtc/AutoBalancer/CMakeFiles/AutoBalancer.dir/AutoBalancer.o] Error 4
make[3]: *** Waiting for unfinished jobs....
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -33,6 +33,7 @@ before_install: # Use this to prepare the system to install prerequisites or dep
   # Define some config vars
   - export CI_SOURCE_PATH=$(pwd)
   - export REPOSITORY_NAME=${PWD##*/}
+  - export ROS_PARALLEL_JOBS="-j2 -l2"
   - echo "Testing branch $TRAVIS_BRANCH of $REPOSITORY_NAME"
   - sudo sh -c 'echo "deb http://packages.ros.org/ros-shadow-fixed/ubuntu precise main" > /etc/apt/sources.list.d/ros-latest.list'
   - wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
@@ -81,13 +82,13 @@ before_script: # Use this to prepare your build for testing e.g. copy database c
   - if [ $BUILDER == rosbuild ]; then rospack profile              ; fi
 script: # All commands must exit with code 0 on success. Anything else is considered failure.
   # for catkin
-  - if [ $BUILDER == catkin ]; then catkin_make -j8 -l8            ; fi
+  - if [ $BUILDER == catkin ]; then catkin_make ${ROS_PARALLEL_JOBS}            ; fi
   - if [ $BUILDER == catkin ]; then export TARGET_PKG=`find build/$REPOSITORY_NAME -name Makefile -print |  sed s@.*/\\\\\([^\/]*\\\\\)/Makefile@\\\1@g` ; fi
   # - if [ $BUILDER == catkin ]; then source devel/setup.sh; export EXIT_STATUS=0; for pkg in $TARGET_PKG; do (roscd $pkg; [ "`find . -iname '*.test'`" == "" ] && echo "[$pkg] No tests ware found!!!"  || find . -iname "*.test" -print0 | xargs -0 -n1 rostest || export EXIT_STATUS=$?;) done; [ $EXIT_STATUS == 0 ] ; fi
   - export EXIT_STATUS=$?
-  - if [ $BUILDER == catkin ]; then catkin_make test --pkg $TARGET_PKG -j8 -l8  ; export EXIT_STATUS=$?; fi
+  - if [ $BUILDER == catkin ]; then catkin_make test --pkg $TARGET_PKG ${ROS_PARALLEL_JOBS}  ; export EXIT_STATUS=$?; fi
   - if [ $EXIT_STATUS != 0 -a $BUILDER == catkin ]; then find build -name LastTest.log -exec echo "==== {} ====" \; -exec cat {} \;  ; fi
-  - if [ $BUILDER == catkin ]; then catkin_make -j8 -l8 install            ; fi
+  - if [ $BUILDER == catkin ]; then catkin_make ${ROS_PARALLEL_JOBS} install            ; fi
   - if [ $BUILDER == catkin ]; then rm -fr devel src build                 ; fi
   - if [ $BUILDER == catkin ]; then source install/setup.bash              ; fi
   - if [ $BUILDER == catkin ]; then export EXIT_STATUS=0; for pkg in $TARGET_PKG; do [ "`find install/share/$pkg -iname '*.test'`" == "" ] && echo "[$pkg] No tests ware found!!!"  || find install/share/$pkg -iname "*.test" -print0 | xargs -0 -n1 rostest || export EXIT_STATUS=$?; done; [ $EXIT_STATUS == 0 ] ; fi

--- a/hironx_ros_bridge/test/test-hironx-ros-bridge.test
+++ b/hironx_ros_bridge/test/test-hironx-ros-bridge.test
@@ -16,5 +16,6 @@
   <param name="joint_states_test/test_duration" value="5.0" />
 
   <!-- test test-name="joint_states_test" pkg="rostest" type="hztest" name="joint_states_test" / -->
-  <test type="test_hironx_ros_bridge.py" pkg="hironx_ros_bridge" test-name="test_hironx_ros_bridge" time-limit="1000" />
+  <test type="test_hironx_ros_bridge.py" pkg="hironx_ros_bridge" test-name="test_hironx_ros_bridge" time-limit="1000"
+        args="-ORBInitRef NameService=corbaloc:iiop:localhost:2809/NameService" />
 </launch>

--- a/hironx_ros_bridge/test/test_hironx_ros_bridge.py
+++ b/hironx_ros_bridge/test/test_hironx_ros_bridge.py
@@ -224,12 +224,12 @@ class TestHiroROSBridge(unittest.TestCase):
         goal = self.goal_LArm()
         for av in [[  25,-139,-157,  45,   0,   0]]:
             goal = self.setup_Positions(goal, [av])
-            self.rarm.send_goal_and_wait(goal)
+            self.larm.send_goal_and_wait(goal)
             # check if tf and current link is same
             rospy.sleep(1)
             now = rospy.Time.now()
             self.listener.waitForTransform("WAIST", "LARM_JOINT5_Link", now, rospy.Duration(1.0))
-            (pos_tf, rot_tf) = self.listener.lookupTransform("WAIST", "RARM_JOINT5_Link", now)
+            (pos_tf, rot_tf) = self.listener.lookupTransform("WAIST", "LARM_JOINT5_Link", now)
             rot_tf = quaternion_matrix(rot_tf)[0:3,0:3]
             pos_c = self.robot.getCurrentPosition('LARM_JOINT5','WAIST')
             rot_c = self.robot.getCurrentRotation('LARM_JOINT5','WAIST')

